### PR TITLE
Tidying Up instructions

### PR DESCRIPTION
**Pull request by Humzini**
Improved flow of install instructions

## Code contributions to the Bitmessage project

- try to explain what the code is about
- try to follow [PEP0008](https://www.python.org/dev/peps/pep-0008/)
- make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
- it should be possible to do a fast-forward merge of the pull requests
- PGP-sign the commits included in the pull request
- You can get paid for merged commits if you register at [Tip4Commit](https://tip4commit.com/github/Bitmessage/PyBitmessage)

If for some reason you don't want to use github, you can submit the patch using Bitmessage to the "bitmessage" chan, or to one of the developers.
## Translations

For helping with translations, please use [Transifex](https://www.transifex.com/bitmessage-project/pybitmessage/). There is no need to submit pull requests for translations.
For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).

### CODE REVIEW

**Comment by g1itch** [comment]
For what purpose this minuses? I don't understand it.

**Comment by PeterSurda** [comment]
Formatting, although there should be a newline between them.

**Comment by g1itch** [comment]
@PeterSurda this just adds minuses, see the [preview](/Humzini/PyBitmessage/blob/afdad30ef3e713c3d541294d46ee6903a917082f/INSTALL.md).

File: INSTALL.md [hunk 1]
```diff
@@ -17,8 +22,10 @@ Here's a list of dependencies needed for PyBitmessage
 - (Fedora & Redhat only) openssl-compat-bitcoin-libs 
 
 ## Running PyBitmessage
-PyBitmessage can be run two ways: straight from source or via a package which
-is installed on your system. Since PyBitmessage is Beta, it is best to run
+PyBitmessage can be run in two ways: 
+-straight from source
```
Comment (PeterSurda) [inline INSTALL.md]: newline missing

File: INSTALL.md [hunk 2]
```diff
@@ -17,8 +22,10 @@ Here's a list of dependencies needed for PyBitmessage
 - (Fedora & Redhat only) openssl-compat-bitcoin-libs 
 
 ## Running PyBitmessage
-PyBitmessage can be run two ways: straight from source or via a package which
-is installed on your system. Since PyBitmessage is Beta, it is best to run
+PyBitmessage can be run in two ways: 
+-straight from source 
+or 
+-via a package which is installed on your system. Since PyBitmessage is Beta, it is best to run
```
Comment (PeterSurda) [inline INSTALL.md]: newline missing

**Comment by PeterSurda** [comment]
it's ok now, I'll merge it tomorrow

**Comment by PeterSurda** [comment]
@omkar1117 well this is markdown, not python.

**Pull request merged**